cli/src/cli_builder.rs

```diff
@@ -134,6 +134,15 @@ impl Default for NewCli<'_> {
                             .required(true)
                             .value_parser(clap::value_parser!(std::net::IpAddr)),
                     ),
+                CliCommand::CertificateCreate => cmd
+                    .mut_arg("cert", |arg| {
+                        arg.value_name("cert-file")
+                            .help("path to PEM-formatted string containing public certificate chain")
+                    })
+                    .mut_arg("key", |arg| {
+                        arg.value_name("key-file")
+                            .help("path to PEM-formatted string containing public certificate chain")
```

Comment:
```suggestion
                            .help("path to a PEM-formatted file containing a public certificate chain")
```

is there some distinction between "key" and "cert" that we'd like to make clear in these help messages?

Reply:
A stupid copy paste error, fixed with b257a04.